plugins/inputs/fritzbox/sample.conf

```diff
@@ -0,0 +1,36 @@
+# Gather fritzbox status
+[[inputs.fritzbox]]
+  ## The devices to query. For each device the corresponding URL including the
+  ## user and passwort needed to login must be set.
+  ## E.g.
+  ## devices = [
+  ##   "http://boxuser:boxpassword@fritz.box:49000/",
+  ##   "http://:repeaterpassword@fritz.repeater:49000/",
+  ## ]
+  devices = [
+  ]
+
+  ## The information to query (see README for further details).
+  ## Hosts info is disabled by default, as it generates an extensive amount
+  ## of data.
+  # device_info = true
+  # wan_info = true
+  # ppp_info = true
+  # dsl_info = true
+  # wlan_info = true
+  # hosts_info = false
+
+  ## Some metric queries are time-consuming and not collected on every query
+  ## cycle. This counter defines how often these low-traffic queries
+  ## are excuted. The default value 30 means, on every 30th query corresponding
+  ## to every 5 minutes (assuming the standard query interval of 10s).
+  ## If this option is set to 1 or below, all metrics are collected on every
+  ## query cycle.
+  # full_query_cycle = 30
```

Comment:
First of all, this section does not explain which specific metrics are affected by this. And frankly, I think it's best to remove this functionality but describe this functionality in the readme by adding an example config with 2 instances of this plugin.

```toml
[[inputs.fritzbox]]
  devices = []

[[inputs.fritzbox]]
  interval = "5m"

  devices = []

  device_info = false
  wan_info = false
  ppp_info = false
  dsl_info = false
  wlan_info = false
  hosts_info = true
```

Reply:
Will rework

Reply:
Adapted via 9ffdb5365d6efaa15f21609dc796ce850ba0b8cb